### PR TITLE
added order show page and authenticated user checkout feature

### PR DESCRIPTION
**Pull request by scottalexandra**

### CODE REVIEW

File: app/models/line_item.rb [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+  def title
+    item.title
+  end
+
+  def description
+    item.description
+  end
+
+  def price
+    item.price / 100
```
Comment (bryceholcomb) [inline app/models/line_item.rb]: we can use the Item#currency method here

File: spec/feature/authenticated_user_spec.rb [hunk 2]
```diff
@@ -121,16 +121,46 @@
 
   it "checkout" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
-    and_return(valid_user)
+                                                    and_return(valid_user)
     click_add_to_cart_link("Breakfast")
     click_link_or_button "Cart:"
     click_link_or_button "Checkout"
+    save_and_open_page
```
Comment (rshea303) [inline spec/feature/authenticated_user_spec.rb]: save_and_open_page still listed in code on #128

File: app/views/orders/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<h1>Your Order</h1>
```
Comment (bryceholcomb) [inline app/views/orders/show.html.erb:1]: Any chance we could reuse the order_items partial here?

File: spec/feature/authenticated_user_spec.rb [hunk 4]
```diff
@@ -121,16 +121,46 @@
...
+      expect(page).to have_content("Bacon and Eggs")
+    end
+    within("#item-description") do
+      expect(page).to have_content("The classic breakfast dish")
+    end
+    within("#item-quantity") do
+      expect(page).to have_content("1")
+    end
+    within("#item-price") do
+      expect(page).to have_content("10.00")
```
Comment (bryceholcomb) [inline spec/feature/authenticated_user_spec.rb]: Are these showing up as $10.00? I see in the view you are using `number_to_currency` so they should have the dollar sign if you want to have the tests reflect that

File: app/views/orders/show.html.erb (hunk 3 above)
Comment (scottalexandra) [inline app/views/orders/show.html.erb:1]: THe show page has unique requirements per the specs, so would like to opt out of the partial in this case.

**Pull request merged**